mindmaps-engine/src/main/java/io/mindmaps/engine/loader/BlockingLoader.java

```diff
@@ -42,34 +43,35 @@
  */
 public class BlockingLoader extends Loader {
 
-    private ExecutorService executor;
-    private Cache cache;
+    private static ConfigProperties prop = ConfigProperties.getInstance();
+
+    private static int repeatCommits = prop.getPropertyAsInt(ConfigProperties.LOADER_REPEAT_COMMITS);
```

Comment:
Just checking but is this config property used everywhere ? If yes then no change is required then if no I would just make this static number here and not in ConfigProperties.